java/client/src/main/java/glide/api/models/commands/geospatial/GeoSearchResultOptions.java

```diff
@@ -42,38 +38,38 @@ public class GeoSearchResultOptions {
     /** Indicates the number of matches the result should be limited to. */
     private final long count;
 
-    /** Indicates if the 'ANY' keyword is used for 'COUNT'. */
+    /** Whether to allow returning as soon as enough matches are found. */
     private final boolean isAny;
 
-    /** Constructor with count only. */
+    /** Define number of search results. */
     public GeoSearchResultOptions(long count) {
         this.sortOrder = null;
         this.count = count;
         this.isAny = false;
     }
 
-    /** Constructor with count and the 'ANY' keyword. */
+    /** Define number of search results <code>ANY</code> option. */
```

Comment:
nit: I think we could just remove the constructor docs since they seem self-explanatory to me, but if we want to keep them this might be a bit clearer
```suggestion
    /** Define number of search results, and whether or not the <code>ANY</code> option should be used. */
```